packages/runtime-dom/src/directives/vOn.ts

```diff
@@ -0,0 +1,36 @@
+const modifierGuards: Record<string, (e: Event) => void | boolean> = {
```

Comment:
```js
const type modifierGuardHandlerFn = (e: Event) => void | boolean;

const modifierGuards: Record<string, modifierGuardHandlerFn> = {...
```

Reply:
I think putting it inline wouldn't harm readability, and I'd prefer fewer lines...

Reply:
This suggestion is only required if we go ahead with declaring the above `guard` variable outside the loop.